sales.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE>
```

Comment:
This is supposed to be \<!DOCTYPE html\> to tell the browser which version of HTML is in the file